presto-yugabyte/src/main/java/io/prestosql/plugin/yugabyte/YugabytePlugin.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.cassandra;
+
+import com.google.common.collect.ImmutableList;
+import io.prestosql.spi.Plugin;
+import io.prestosql.spi.connector.ConnectorFactory;
+
+/*
+ * This class is the plugin.
+ */
+public class YugabytePlugin
+        implements Plugin
+{
+    @Override
+    public Iterable<ConnectorFactory> getConnectorFactories()
+    {
+        return ImmutableList.of(new CassandraConnectorFactory());
```

Comment:
Plugin can provide zero or more connector factories and the connector factories are distinguished by `io.prestosql.spi.connector.ConnectorFactory#getName` (which is typically equal to plugin name, but doesn't have to be).

Currently, the new plugin exports `cassandra` connector factory.
You need to parameterize `CassandraConnectorFactory`.
See `io.prestosql.plugin.hive.HiveConnectorFactory#name` for example